internal/constants/consts.go

```diff
@@ -37,3 +37,11 @@ const (
 	UserDeviceAPIIntegrationStatusDuplicateIntegration  string = "DuplicateIntegration"
 	UserDeviceAPIIntegrationStatusAuthenticationFailure string = "AuthenticationFailure"
 )
+
+// PrivilegeIDs for claims.
+const (
+	NonLocationData int64 = 1
```

Comment:
Maybe we should pull these out to the shared repo.

Reply:
100% agree since they are also used in device-data.